### PR TITLE
Workflow Added GeoIP Lookup with Slack

### PR DESCRIPTION
**Pull request by tylerslijboom-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [x] Workflow verified as working in a test
- [x] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [x] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Workflow includes a detailed Description
- [x] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [x] Triggers, Decisions, and other workflow steps include descriptions where needed
- [x] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [x] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [x] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [x] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<img width="459" alt="workflow" src="https://user-images.githubusercontent.com/57681561/79497060-15579d80-7ff5-11ea-8d81-254d1459c842.png">


#### Artifact

<img width="300" alt="artifact" src="https://user-images.githubusercontent.com/57681561/79497092-20aac900-7ff5-11ea-97e4-fc71fb7f1f92.png">

```

### CODE REVIEW

File: workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+extension: workflow
+products: ["insightconnect"]
+name: GeoIP_Lookup_with_Slack
+title: "GeoIP Lookup with Slack"
+description: This workflow accepts a Slack command, and performs a GeoIP lookup using the IPStack
```
Comment (jmcadams-r7) [inline workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml:5]:
```suggestion
description: This workflow accepts a Slack command, and performs a GeoIP lookup using the IPStack plugin
```

File: workflows/GeoIP_Lookup_with_Slack/GeoIP Lookup with Slack.icon [hunk 2]
```diff
@@ -0,0 +1,884 @@
...
+          "96b62e72-3067-4505-9efc-be3cce6fe8b7": {
+            "nodeId": "96b62e72-3067-4505-9efc-be3cce6fe8b7",
+            "name": "Print GeoIP details to Slack",
+            "type": "action_chatops",
+            "continueOnFailure": false,
+            "isDisabled": false,
+            "parameters": {
+              "input": {
+                "attachments": [],
+                "message": ":notebook_with_decorative_cover: GeoIP Lookup for `{{[6f80dc1f-9a48-4fbd-bfa0-06f2a86a6c04].[content]}}` :notebook_with_decorative_cover:\n_WhoIs_\n> Address: {{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[address]}}\n> City: {{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[city]}}\n> Region: {{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[region_name]}}\n> Country: {{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[country_name]}}\n> Zip Code: {{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[zip_code]}}\n> Co-ordinates: `({{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[latitude]}},{{[ebd2bfab-ad02-4e0d-8db9-1c1b5bf5cde7].[longitude]}})`\n\n",
```
Comment (jmcadams-r7) [inline workflows/GeoIP_Lookup_with_Slack/GeoIP Lookup with Slack.icon:446]: 'Coordinates' is one word.

File: workflows/GeoIP_Lookup_with_Slack/help.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+This workflow accepts a Slack command containing an IP Address, looks up the IP in the IP Stack Database, and returns
+details about where the IP Address is located on the globe.
+
+Sample Slack Trigger Commands:
+
+`@Slack bot !geoip <IP Address>``
+
+# Key Features
+
+* Location Lookup on an IP Address from Slack
```
Comment (jschipp-r7) [inline workflows/GeoIP_Lookup_with_Slack/help.md:12]:
```suggestion
* Geo location lookup on an IP Address from Slack
```

File: workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml (hunk 1 above)
Comment (jschipp-r7) [reply workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml:5]: Update the description key in the workflow file to match as well

File: workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [data_enrichment, threat_detection_and_response]
+  keywords: [data_enrichment, threat_detection_and_response, chat, chatops]
```
Comment (jschipp-r7) [inline workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml:12]:
```suggestion
  keywords: [geoip, location, chat, chatops]
```

File: workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+version: 1.0.0
+vendor: rapid7
+support: rapid7
+status: []
+hub_tags:
+  use_cases: [data_enrichment, threat_detection_and_response]
+  keywords: [geoip, location, chat, chatops]
+  features: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-workflows/tree/master/workflows/Unshorten_URL_From_Slack
```
Comment (jschipp-r7) [inline workflows/GeoIP_Lookup_with_Slack/workflow.spec.yaml:15]: This is the wrong URL

**Comment by tylerslijboom-r7** [comment]
workflow

**Pull request merged**